proxy/src/fully_qualified_authority.rs

```diff
@@ -189,18 +197,16 @@ mod tests {
         assert_eq!("name.namespace.svc.cluster.local",
                    local("name.namespace.svc", "namespace"));
         external("name.namespace.svc.cluster", "namespace");
+        external("name.namespace.svc.cluster", "namespace");
         assert_eq!("name.namespace.svc.cluster.local",
                    local("name.namespace.svc.cluster.local", "namespace"));
 
         // Fully-qualified names end with a dot and aren't modified except by removing the dot.
-        assert_eq!("name",
-                   local("name.", "namespace"));
-        assert_eq!("name.namespace",
-                   local("name.namespace.", "namespace"));
-        assert_eq!("name.namespace.svc",
-                   local("name.namespace.svc.", "namespace"));
-        assert_eq!("name.namespace.svc.cluster",
-                   local("name.namespace.svc.cluster.", "namespace"));
+        external("name.", "namespace");
+        external("name.namespace.", "namespace");
+        external("name.namespace.svc.", "namespace");
+        external("name.namespace.svc.cluster.", "namespace");
+        external("name.namespace.svc.acluster.local.", "namespace");
```

Comment:
looks like we lost a tests for: `name`, `name.namespace`, `name.namespace.svc`, and `name.namespace.svc.cluster`. Why?

Reply:
oh, i see, i was misreading the test. carry on.